### ISSUE TITLE
Add a Labs page

### ISSUE THREAD

**Issue by ondrae** [codeforamerica/brigade#342]
## Civic Tech Movement Lab
Add a page to the Brigade site to show off our various experiments in infrastructure for the civic tech movement.

The intent of the page is to be a home for several separate projects. Also, to win support for more staff / help on building these experiments out.

### Launched Projects
* The CfAPI - http://codeforamerica.org/api
* The Brigade website - http://www.codeforamerica.org/brigade/
* The Civic Issue Finder - http://www.codeforamerica.org/geeks/civicissues

### In Progress projects
* Civic Tech Projects search - http://www.codeforamerica.org/brigade/projects - http://www.codeforamerica.org/brigade/Code-for-San-Francisco/projects/
* Brigade Project monitor - http://brigade-projectmonitor.herokuapp.com/ - http://brigade-projectmonitor.herokuapp.com/Chi-Hack-Night/
* Brigade Sign Up Form - http://www.codeforamerica.org/brigade/signup/
* Attendance Tool - https://github.com/codeforamerica/people-app#checkins
* Civic Tech Numbers - http://www.codeforamerica.org/brigade/numbers/

### Idea Projects
* People Page with badges - https://github.com/codeforamerica/brigade/issues/341
* Use the CfAPI for another networked movement - Public Media - http://publicmedia-website.herokuapp.com/brigade/ - https://publicmedia-issues.herokuapp.com/geeks/civicissues/widget - http://publicmedia-cfapi.herokuapp.com/api/organizations

**Comment by ondrae** [comment]
We should probably attach each of these projects to our goals and metrics ...

**Comment by mollymcleod** [comment]
Would it make sense to use similar categories/labels as the product stages we're thinking about for the apps/work page on the CfA site? Alpha/Beta/Prototype etc. cc @cydharrell

**Comment by ondrae** [comment]
Yeah, I'd like to use the same categories as we decide for other projects.  Its important for me in a few different places.

We recently built support for people to [tell us the status](https://github.com/codeforamerica/cfapi#civicjson) of their projects.

We also want people to [easily search](http://codeforamerica.org/brigade/projects) along those statuses.

I've changed it to Launched, Beta, and Alpha, but would like more advice.
